core/domain/voiceover_domain.py

```diff
@@ -316,3 +316,160 @@ def create_empty(
             language_accent_code=language_accent_code,
             voiceovers_mapping={},
             automated_voiceovers_audio_offsets_msecs={})
+
+
+class ExplorationVoiceArtistsLinkDict(TypedDict):
+    """Dict type for Exploration Voice Artists object."""
+
+    exp_id: str
+    content_id_to_voiceovers_mapping: Dict[
+        str, Dict[str, Tuple[str, state_domain.VoiceoverDict]]]
+
+
+class ExplorationVoiceArtistsLink:
+    """Domain object for linking Exploration with voice artists."""
+
+    def __init__(
+            self,
+            exp_id: str,
+            content_id_to_voiceovers_mapping: Dict[
+                str, Dict[str, Tuple[str, state_domain.VoiceoverDict]]
+            ]
+    ) -> None:
+        """Construct a ExplorationVoiceArtistsLink domain object.
+
+        Args: 
+            exp_id: str. ID of exploration.
+            content_id_to_voiceovers_mapping: Dict. A dict consisting 
+                of content IDs as key and other dicts as value.
+                This other dict consists of a bunch of languages
+                (in which voiceovers of content is made) as keys
+                and a two-tuple as values. First value of tuple
+                is voice artist ID and second value is VoiceOverDict.
```

Comment:
Let's copy over the comment from [here](https://github.com/oppia/oppia/blob/7e06003db0f867ea94ee07af459f6a3b616bcb00/core/storage/voiceover/gae_models.py#L393C5-L396C29).